### ISSUE TITLE
Package naming

### ISSUE THREAD

**Issue by Jonesus** [Jonesus/pony-stubs#6]
Should the package be named `pony-stubs`, `types-pony` or something else?

@fluffy-critter suggested that
> Also, just as a note, a more standard name for the types package would be types-pony.

at https://github.com/ponyorm/pony/issues/479#issuecomment-1100930108

But as far as I could find, this PEP (https://peps.python.org/pep-0561/#type-checker-module-resolution-order) suggests that type-only modules should be named `<package_name>-stubs`. Personally I don't have an opinion other than that we go as standard as possible.

@fluffy-critter can you find another PEP that would propose the naming you suggested, or is it more based on what is commonly done?

**Comment by fluffy-critter** [comment]
I'm basing it on the various types packages I've seen, for example `types-python-dateutil`, `types-requests`, and `types-atomicwrites`. I wasn't aware of any PEP-provided naming scheme, and I hadn't ever come across a `<package>-stubs` package before either although it appears there are quite a few with that naming convention as well (e.g. `lxml-stubs`).

**Comment by Jonesus** [comment]
Thanks, now that I look at MyPy docs for example, [they claim that](https://mypy.readthedocs.io/en/stable/installed_packages.html#:~:text=As%20many%20packages%20don%E2%80%99t%20support%20type%20checking%20yet%2C%20you%20may%20also%20have%20to%20install%20a%20separate%20stub%20package%2C%20usually%20named%20types%2D%3Clibrary%3E)

> As many packages don’t support type checking yet, you may also have to install a separate stub package, usually named `types-<library>`.

but afterwards [they suggest](https://mypy.readthedocs.io/en/stable/installed_packages.html#:~:text=If%20the%20package%20is%20stub%2Donly%20(not%20imported%20at%20runtime)%2C%20the%20package%20should%20have%20a%20prefix%20of%20the%20runtime%20package%20name%20and%20a%20suffix%20of%20%2Dstubs) 

> If the package is stub-only (not imported at runtime), the package should have a prefix of the runtime package name and a suffix of -stubs

which might be a bit confusing if not contradictory :smile: but I guess we'll keep the name as is for now, at least for Pyright it also enables [automatic stub package resolution](https://github.com/microsoft/pyright/blob/main/docs/import-resolution.md#:~:text=For%20a%20given%20package%2C%20try%20to%20resolve%20first%20using%20a%20stub%20package.%20Stub%20packages%2C%20as%20defined%20in%20PEP%20561%2C%20are%20named%20the%20same%20as%20the%20original%20package%20but%20with%20%E2%80%9C%2Dstubs%E2%80%9D%20appended.)

**Comment by fluffy-critter** [comment]
Yeah, sounds good to me. Weird that this inconsistency happened, but that's a larger Python ecosystem issue and not something specific to this package. :)
